docs/configuration/generating-docs.rst

```diff
@@ -41,6 +42,15 @@ You can use the :class:`~cosmos.operators.DbtDocsS3Operator` to generate and upl
         bucket_name="test_bucket",
     )
 
+    generate_dbt_freshness_aws = DbtFreshnessS3Operator(
+        task_id="generate_dbt_freshness_aws",
+        project_dir="path/to/jaffle_shop",
+        profile_config=profile_config,
+        # docs-specific arguments
+        connection_id="test_aws",
+        bucket_name="test_bucket",
+    )
+
```

Comment:
I just realised we're adding the code manually in this doc code.

WDYT if we added a reference to the example DAG, similar to what we did in:
https://github.com/astronomer/astronomer-cosmos/blob/6225db8570fbfd92a81e4c497b5b1f651ac0ae6e/dev/dags/basic_cosmos_dag.py#L24
https://github.com/astronomer/astronomer-cosmos/blob/188fe56994e29037eda1777974faa45a183ebc1f/docs/getting_started/execution-modes.rst?plain=1#L58-L61

This way, the documentation will be up-to-date if the operator changes without additional effort.